TACC-Cloud/hazmapper
# task/wg-75-project-listing-main

## Overview: ##
Create React project listing.

## PR Status: ##


## Related Jira tickets: ##

* [WG-75](https://tacc-main.atlassian.net/browse/WG-75)

## Summary of Changes: ##
- Installed core-styles lts and added to index.tsx
- imported styles into index.html head
- Created project listing component and added component to main menu page to view
- created hooks for getting projects 

- created types for projects responses, ds project responses, and ds project collections (raw response)

## Testing Steps: ##
1. Go to localhost:4200 and make sure your projects display
2. Links and button do not work 

## UI Photos:
<img width="915" alt="Screenshot 2024-01-12 at 4 54 14 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/96220951/325ffdb1-1dd1-45e7-b56b-55c7646bbc31">

## Notes: ##
TODOs
- need to create a hook that uses a query function to combine ds response and tapis project response
- need to install icons into project to show the edit and trash icons for each map project
